docs/markdown/Contributing.md

```diff
@@ -547,7 +547,7 @@ This piece of code would never converge. Every Meson run would change
 the value of the option and thus the output you get out of this build
 definition would be random.
 
-Meson does not permit this by forbidding these sorts of covert
+Meson does not permit this by forbidding these sorts of convert
```

Comment:
???

Reply:
That was originally correct.

Reply:
Right, that is a false positive. ;) Automated spellcheckers aren't perfect. :D